test/test_scanner.py

```diff
@@ -184,6 +184,42 @@ def _file_test(self, url, filename, package, version):
                 # From NVD : CVE-2017-1000101 affects 7.35.0 to 7.55.0
                 [],
             ),
+            (
+                "test-curl-libcurl.so.7.34.0.out",
+                "curl",
+                "7.34.0",
+                [
+                    "CVE-2019-3823",
+                    "CVE-2018-14618",
+                    "CVE-2017-1000101",
+                ],  # CVE-2017-1000101 needs supplemental data
```

Comment:
If CVE-2017-1000101 is now being detected correctly, we don't need this comment any more.

Reply:
@terriko you can leave these PR's for now because I'll have to produce tests for the new Checker Based Approach

Reply:
Ok.  Please close any you don't want reviewed!